io.github.punesemu.puNES.yml

```diff
@@ -0,0 +1,88 @@
+app-id: io.github.punesemu.puNES
+runtime: org.kde.Platform
+runtime-version: "5.15-22.08"
+sdk: org.kde.Sdk
+command: punes
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=ipc
+  - --device=all
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=/run/udev:ro
```

Comment:
I don't think I've seen other apps use it, including Steam. Are you sure it's needed?

Reply:
`--device=all` is required to access recognized gamepads.
`--share=network` you are right, this is not necessary, I do not use the network.
`--socket=pulseaudio` it's required for audio.
`--filesystem=/run/udev:ro` I use libudev to check and recognize devices connected to USB ports and without this  libudev don't find nothing.

Reply:
Yeah, I meant only the last line. Thanks.